ANSWERS.md

```diff
@@ -0,0 +1,5 @@
+## Questions
+
+1. DOM - Document Object Model. 
```

Comment:
I would have liked to see more elaborate answers. Each of these topics can get pretty complex.